_stylesheet.scss

```diff
@@ -247,11 +247,21 @@ $dock_style_modes: [null, shrink, extended, extended-shrink];
     }
 }
 
+.overview-tile {
+    margin: 0px;
+    padding: 0px;
+    border: 0px;
+}
+
```

Comment:
How is this related to the app icons?

It would be worth a comment on the commit.